creator-node/src/routes/users.js

```diff
@@ -170,4 +170,24 @@ module.exports = function (app) {
 
     return successResponse({ clockValue })
   }))
+  /**
+   * Returns latest clock value stored in CNodeUsers entry given wallet, or -1 if no entry found
+   */
+  app.get('/users/batch_clock_status', handleResponse(async (req, res) => {
+    // TODO: Limit max size
+    const walletPublicKeys = req.query.wallet_public_key // array
+    const cnode_users = await models.CNodeUser.findAll({
+      where: {
+        walletPublicKey: {
+          [models.Sequelize.Op.in]: walletPublicKeys
+        }
```

Comment:
add `raw: true` and it returns raw json instead of a sequelize object

Reply:
got it functioning as is, gonna leave this